setup/functions.py

```diff
@@ -1198,3 +1207,11 @@ def check_affinity():
     except Exception as e:
         announce(f"❌ Error connecting to Kubernetes: {e}")
         return False
+
+def is_standalone_deployment(ev: dict) -> bool:
+    """
+    Returns true if it is a standalone deployment
+    """
+    if int(ev.get("control_environment_type_standalone_active", 0)) == 1:
+        return True
+    return False
```

Comment:
This can be simplified
```suggestion
    return int(ev.get("control_environment_type_standalone_active", 0)) == 1
```